src/bin/xmoji/filewatcher.c

```diff
@@ -1,4 +1,6 @@
+#ifdef __linux__
```

Comment:
Maybe just define the glibc-specific _DEFAULT_SOURCE instead? IMHO better than checking for Linux, glibc might be used on a different OS and Linux might be used with a different libc...

The effect should be the same, _DEFAULT_SOURCE causes glibc to just "expose everything", and we'd rely on other libc implementations to do the same when *no* feature test macro is defined...

Reply:
Using `_DEFAULT_SOURCE` is fine with me, that symbol doesn't mean anything on NetBSD.